qmt/geometry/property_map.py

```diff
@@ -48,21 +45,19 @@ def __call__(self, x):
 class MaterialPropertyMap(PropertyMap):
     """Map points in the simulation domain to material properties of parts containing the points.
 
-    :param PartMap part_map: Function that takes a spatial location and maps it to a part
-                             identifier.
+    :param PartMap part_map: Function that takes a spatial location and maps it to a part identifier.
     :param dict part_materials: Dict mapping from part identifier to a material name.
     :param qmt.Materials mat_lib: Materials library used to look up the material properties.
     :param str prop_name: Name of the material property to be retrieved for each part.
     :param eunit: Energy unit, passed to `mat_lib.find()`.
-    :param fill_value: Value to be filled in places where there is no part or the part does not have
-        a material or the material does not have the property `prop_name`. The default behavior
-        `fill_value='raise'` is to raise a KeyError in these cases.
+    :param fill_value: Value to be filled in places where there is no part or the part does not have a material or the material does not have the property `prop_name`. The default behavior `fill_value='raise'` is to raise a KeyError in these cases.
```

Comment:
Minor nitpick: why are these docstrings crammed into ultra-long lines now?

Reply:
I like to have the text editor's textwrap take care of this. But if you don't like that we can revert back.

I'm going to merge first b/c I need this to do stuff on the qms side. Maybe we should standardize on this some time :P